website/sidebars.json

```diff
@@ -5,7 +5,7 @@
       "docker-development",
       "native-installation"
     ],
-    "Recipes": ["django-react-interop"],
+    "Recipes": ["django-react-interop", "building-the-frontend"],
```

Comment:
we should maybe start a multi-line array ?

Reply:
That's `prettier`'s job 😇

Reply:
I'm not sure the CI run prettier in the `website` directory.

Reply:
Indeed, and we should probably add it. I was just saying in this instance I formatted with `prettier` and it does not put a newline here

Reply:
Done #893